docs/search relevance/test_1.md

```diff
@@ -0,0 +1,72 @@
+# Test 1
+## Candidates
+Four initial candidates were assembled to get a feel for the effects of each lever we can pull. They were tested explicitly with internal users. Using the json query format, they look like this:
+
+**just boosting**
+```
+{
+  "query": {
+    "multi_match" : {
+      "query" : "oil painting of urine",
+      "fields" : ["*", "subjects*^4", "genres*^4", "title^3"], 
+      "type": "cross_fields"
+    }
+  }
+}
+```
+
+**broader boosting**
+```
+{
+  "query": {
+    "multi_match" : {
+      "query" : "oil painting of urine",
```

Comment:
A good example of why this work is important:

https://api.wellcomecollection.org/catalogue/v2/works?query=urine%20painting%20of%20oil
https://api.wellcomecollection.org/catalogue/v2/works?query=oil%20painting%20of%20urine